File: C++/max_consecutive_1.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin >> n;
    vector<int> nums(n);
    for (int i=0; i<n; i++){
        cin >> nums[i];
    }
    int max = 0, count = 0;
    for(int i : nums){
        if(i == 1){
            count++;
            if(count > max){
                max = count;
            }
        } else {
            count = 0;
        }
    }
    cout << max;
}

File: Python/max_consecutive_1.py
n = int(input())
nums = []
for _ in input().split():
    nums.append(int(_))

max = 0
count = 0
for i in nums:
    if i == 1:
        count += 1
        if count > max:
            max = count
    else:
        count = 0

print(max)